drizzle.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "drizzle-kit";
+
+export default defineConfig({
+  dialect: "postgresql",
+  schema: "./src/db/schema/*",
+  // Converts js-idiomatic `e.firstName` to db-idiomatic `first_name` columns
+  casing: "snake_case",
```

Comment:
Bit of a shame it doesn't pick this up from `db/index.ts` but it is what it is. A bit too magical, it would appear.